src/extensions/surveys.ts

```diff
@@ -585,7 +595,7 @@ export const createMultipleChoicePopup = (posthog: PostHog, survey: Survey, ques
         </div>
         <div class="bottom-section">
         <div class="buttons">
-            <button class="form-submit auto-text-color" type="submit" disabled>${
```

Comment:
I'm adding setting the `disabled` attribute manually below instead so it doesn't leave a `null` attribute in the HTML

Reply:
q: why will it leave a null attribute? We can conditionally add or not add it, can't we?

Much prefer it being in place as otherwise things become harder to debug as there are all these other places modifying the DOM.

Reply:
Ok it looks like empty string works here and won't leave anything, will update